### ISSUE TITLE
How to avoid buffer error in Clipper

### ISSUE THREAD

**Issue by AlejandroSdeL** [joncampbell123/dosbox-x#3206]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar question(s) already exists?

- [X] I have searched and didn't find any similar issues.

### Question

Hi.  I have an old Clipper program, that runs fine in Windows 32 bits.  When it runs in DB-X it gives en error 8, which means "BUFFER ERROR. The system ran out of database and/or index buffer handles".  I noticed that in Windows 32 bits I have in `CONFIG.NT` a line saying `BUFFERS=30` which seems cannot be included in section `[config]` of the DB-X configuration file.  Is there a way to avoid the buffers error?  Thanks.

**Comment by AlejandroSdeL** [comment]
Hi.  I found that DB-X has a command `BUFFERS` that can be used to display and increase the number of DOS buffers.  You can try `BUFFERS /help` at the DB-X prompt for further information.  Regarding the DB-X configuration file, I added `BUFFERS=30` in section `[config]`, which did **not** increase the number of buffers.  It seems that currently the number of buffers cannot be specified in the configuration file, but there is a convenient way to change it using the `BUFFERS` command.  Remember that you can issue this or any other DB-X commands when you start the emulation.

**Comment by AlejandroSdeL** [comment]
Hi.  In my setup the problem appears while indexing a large file.  So I searched the web looking for that kind of problem in Clipper, and I noticed that other people faced the same problem without using DB-X.  I guess there is something wrong in Clipper that triggers the error under certain specific (not completely known) conditions.
I found a number of suggestion to avoid the problem, such as
1. splitting the file before INDEXing,
2. using REINDEX instead of INDEX, and
3. creating an empty file, then the index(es) for this new file, and finally APPENDing the original file into the newly created file.

I did not try option 1 because I thought it was too involved.  Option 2 did not work for me.  Option 3 worked fine.
